Packages/com.unity.inputsystem/InputSystem/Devices/Mouse.cs

```diff
@@ -319,5 +319,33 @@ void IInputStateCallbackReceiver.OnStateEvent(InputEventPtr eventPtr)
         {
             OnStateEvent(eventPtr);
         }
+
+        internal static unsafe bool MergeForward(InputEventPtr currentEventPtr, InputEventPtr nextEventPtr)
+        {
+            if (currentEventPtr.type != StateEvent.Type || nextEventPtr.type != StateEvent.Type)
+                return false;
+
+            var currentEvent = StateEvent.FromUnchecked(currentEventPtr);
+            var nextEvent = StateEvent.FromUnchecked(nextEventPtr);
+
+            if (currentEvent->stateFormat != MouseState.Format || nextEvent->stateFormat != MouseState.Format)
+                return false;
+
+            var currentState = (MouseState*)currentEvent->state;
+            var nextState = (MouseState*)nextEvent->state;
```

Comment:
Mouse has no fixed memory layout so the assumption of state corresponding to `MouseState` isn't guaranteed to be true. Think this one has to be on `FastMouse` for now.

Reply:
Will fix

Reply:
Fixed